app/__test__/unit.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { Generator } from "../utils";
+
+test('returns instance of Generator', () => {
```

Comment:
Perhaps it's better to use `describe` to group related tests and `it` for test cases.

Eg:
```javascript
describe('Generator class', () => {
  it('should return the next ID with the right step', () => {
      const generator = new Generator(0, 10, 2);
      const id1 = generator.getId();
      const 1d2 = generator.getId(); 
      expect(id1).toEqual(2);
      expect(id2).toEqual(4);
  });
});
```